keps/sig-network/563-dual-stack/README.md

```diff
@@ -1299,3 +1311,146 @@ This capability will move to stable when the following criteria have been met.
 * testing ingress controller infrastructure with updated dual-stack services
 * dualstack tests run as pre-submit blocking for PRs
+
+
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [X] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name: IPv6DualStack
+    - Components depending on the feature gate: kube-apiserver, kube-controller-manager, kube-proxy, and kubelet
+  - [ ] Other
+    - Describe the mechanism:
+    - Will enabling / disabling the feature require downtime of the control
+      plane?
+    - Will enabling / disabling the feature require downtime or reprovisioning
+      of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled).
+
+* **Does enabling the feature change any default behavior?**
+  No. Pods and Services will remain single stack until cli flags have been modified
+  as described in this KEP. Once modified, existing and new services will remain
+  single stack until user requests otherwise. Pods will become dual-stack however
+  it will maintain the same ipfamily used in before enabling feature flag.
+
+* **Can the feature be disabled once it has been enabled (i.e. can we roll back
+  the enablement)?**
+  Yes.
+
+* **What happens if we reenable the feature if it was previously rolled back?**
+  Similar to enable it the first time on a cluster.
+
+* **Are there any tests for feature enablement/disablement?**
+  The feature is being tested using integration tests with gate on/off. The
+  tests can be found here: https://github.com/kubernetes/kubernetes/tree/master/test/integration/dualstack
+
+  The feature is being tested on -some of - the cloud providers and kind.
+   1. https://testgrid.k8s.io/sig-network-dualstack-azure-e2e. This has all dualstack tests on azure.
+   2. kind dual-stack iptables: https://testgrid.k8s.io/sig-network-kind#sig-network-kind,%20dual,%20master
+   3. kind dual-stack ipvs: https://testgrid.k8s.io/sig-network-kind#sig-network-kind,%20ipvs,%20master
+
+### Rollout, Upgrade and Rollback Planning
+
+* **How can a rollout fail? Can it impact already running workloads?**
+  Try to be as paranoid as possible - e.g., what if some components will restart
+   mid-rollout?
+
+* **What specific metrics should inform a rollback?**
+  N/A
+
+* **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
+  Describe manual testing that was done and the outcomes.
+  Longer term, we may want to require automated upgrade/rollback tests, but we
+  are missing a bunch of machinery and tooling and can't do that now.
+
+* **Is the rollout accompanied by any deprecations and/or removals of features, APIs, 
+fields of API types, flags, etc.?**
+  Even if applying deprecation policies, they may still surprise some users.
+
+### Monitoring Requirements
+
+* **How can an operator determine if the feature is in use by workloads?**
```

Comment:
Maybe paste a kubectl command that shows how many services are using `ipFamilyPolicy` != (Single || "")

Reply:
Do we want to add a metric for number of services broken down by `ipFamilyPolicy` ?

Reply:
I understand. My point is the existence of a service does not mean it is used by a workload. Services are not like PVs where they have a relation to a PVC/node (attach)/Pod (mount) they are a stand a lone object.

Reply:
I interpret this question more broadly - "how can a cluster admin know if the feature is in use".  To me that means "are there services actively using or trying to use dual-stack"